src/testproblem.jl

```diff
@@ -0,0 +1,23 @@
+using Random, JuMP
```

Comment:
We should include the test scripts in a different folder: 
`test/`

Reply:
Okay.